packages/App/src/components/ButtonApp.tsx

```diff
@@ -1,19 +1,59 @@
 import React, {FC} from 'react';
-import {TouchableOpacity, Text} from 'react-native';
+import {TouchableOpacity, Text, StyleSheet, Image} from 'react-native';
 
 interface Props {
+  disabled?: boolean;
   label: string;
   styleBtn?: any;
   styleTxt?: any;
   onPress?: () => void;
+  image?: any;
 }
 
-const ButtonApp: FC<Props> = ({label, styleBtn, styleTxt, onPress}) => {
+const ButtonApp: FC<Props> = ({
+  label,
+  styleBtn,
+  styleTxt,
+  onPress,
+  disabled,
+  image,
+}) => {
   return (
-    <TouchableOpacity style={styleBtn} onPress={onPress}>
-      <Text style={styleTxt}>{label}</Text>
+    <TouchableOpacity
+      style={styleBtn ? styleBtn : styles.btn}
+      onPress={onPress}
+      disabled={disabled}>
+      <Image source={image} style={styles.img} />
+      <Text style={styleTxt ? styleTxt : styles.text}>{label}</Text>
```

Comment:
Do not use truncated names. 
```suggestion
      <Text style={styleText ? styleText : styles.text}>{label}</Text>
```